model/CMakeLists.txt

```diff
@@ -75,6 +76,7 @@ foreach (xlen IN ITEMS 32 64)
                 "riscv_insts_zbkb.sail"
                 "riscv_insts_zbkx.sail"
                 "riscv_insts_zicond.sail"
+                "riscv_insts_zimop.sail"
                 ${vext_srcs}
                 "riscv_insts_zicbom.sail"
                 "riscv_insts_zicboz.sail"
```

Comment:
Shouldn't we put these new files right at the end so they have lowest precedence?